libs/fn/src/lib/cdk/focusable-list/focusable-list.directive.ts

```diff
@@ -0,0 +1,43 @@
+import { AfterViewInit, ContentChildren, Directive, Input, QueryList } from '@angular/core';
+import { map, startWith, takeUntil, tap } from 'rxjs/operators';
+import { FocusableItem, FocusableListService } from './focusable-list.service';
+import { FocusableItemDirective } from './focusable-item.directive';
+import { DestroyedBehavior } from '../common-behaviors/destroyed-behavior';
+
+@Directive({
+    selector: '[fnFocusableList]',
+    exportAs: 'fnFocusableList',
+    providers: [FocusableListService, DestroyedBehavior]
+})
+export class FocusableListDirective implements AfterViewInit {
+    @Input()
+    navigationDirection: 'horizontal' | 'vertical' = 'vertical';
+
+    @Input()
+    contentDirection: 'ltr' | 'rtl' | null = 'ltr';
```

Comment:
Other components are using fundamental-ngx's RtlService, it would be nice for directionality to come automatically and not as an input

Reply:
Yes, just we have not yet decided how we will gonna handle directionality issue globally on fn

Reply:
service was is pretty usable as in my opinion.

Reply:
Yes, but that exists on core side yet, needs to be moved and somehow docs adapted to it too